examples/roll.py

```diff
@@ -0,0 +1,75 @@
+"""An example of a simple CLI for rolling dice.
+
+The example demonstrates the usage of a typed argument
+accepting a list of custom objects (instances of `Die`).
+
+Usage:
+    $ python roll.py 1d6
+    3
+    $ python roll.py 2d6
+    2 5
+    $ python roll.py 1d6 2d3
+    6
+    2 1
+"""
+
+import re
+import random
+from typing import List
+
+from candies.cli import cli, Arg
+
+
+class Die:
+    """A die to roll."""
+
+    def __init__(self, sides: int, rolls: int):
+        self._sides = sides
+        self._rolls = rolls
+
+    def roll(self) -> List[int]:
+        """Rolls the die the required amount of times."""
```

Comment:
I'm not quite sure, but seems like there should be a blank line. Or is there a rule to not to put one for simple cases like that?

Reply:
I prefer not to put a blank line in a case if the body of a function consists of a single line. Not sure whether there are any rules though.